webapp/actions/integration_actions.jsx

```diff
@@ -33,6 +33,20 @@ export function loadIncomingHooks(complete) {
     );
 }
 
+export function loadIncomingHooksForTeam(teamId, complete) {
+    IntegrationActions.getIncomingHooks(teamId, 0, 10000)(dispatch, getState).then(
```

Comment:
Should we now consider the use of per page limit for the request? 10k per page might be too big to handle for use cases with large number of hooks.